### PR TITLE
remove json minify; require fhir_models >= 4.0.0

### PR DESCRIPTION
**Pull request by Shaumik-Ashraf**
Addresses ticket [DVCI-132](https://jira.mitre.org/browse/DVCI-132).

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -122,9 +121,6 @@ GEM
       concurrent-ruby (~> 1.0)
     jbuilder (2.11.2)
       activesupport (>= 5.0.0)
-    json (2.5.1)
```
Comment (mickohanlon23) [inline Gemfile.lock:125]: Maybe similarly remove lines 24-26 in test_app/Gemfile.lock

**Review by mickohanlon23 — APPROVED** [review]

**Review by radamson — APPROVED** [review]
Code looks good. Created and validated new health cards using this branch with no minimization issues detected.

<details>
<summary> Health Card Examples </summary>

inez_qr

**inez.smarth-health-card (Cannot upload directly to GitHub)**

```json
{"verifiableCredential":["eyJ6aXAiOiJERUYiLCJhbGciOiJFUzI1NiIsImtpZCI6ImFma0kyM1VMeFBKOTZoTEZvM2o0V2M1TTRuVFE0WG9FczFUZk92Q09DTUUifQ.fVHLbsIwEPyX7TUPOyBacmx74Va1tJeKg3EW4sqxI9uJCij_3nWgAtGHb2vPjGdmD6C8hxLqENoyz7WVQtfWh3LCGIMEzHoDJZ8VUzYt2JQl0EsoDxB2LUL5PtI88XwjXKhR6FBnUrjK3xyHNA4k8zdO2l5VfP4vRjVNZ9ReBGUNrBKQDis0QQn90q0_UIZoaVMr94bOR0wJ04xlnETj7X1nKo1n2yCt1sSKyARIyO0oCyl0Wr86TQCH3nZOYhkr-B6igBENHrGiUZpo8GxrEWW2qkcTK1kY3MNqIJtrRTkeRYhf8vndJGXzlF8qLo92nigYuYBhSH41wa9M-CBC58ccTasxYGy4F1Iqgw-2GjHSVspsR6t-5wM25x3X-jazbpvHanKvqlz2nyQgRyYUvIBhNSTQnlyRmMMNOjTx98tqCGSl7Nz4FHMuVTNKsIKnbJby2c-si8tVDrGlK8BpWQOdLw.hwGLogER5CsZaa3R2503WUpY9DQ-hhRwo9h72_YsNYa5eJNplaqsHmXGOL2Ieu9gthlFIYZog0yutpc6OmOIAA"]}
```

</details>

**Comment by radamson** [comment]
Side Note:

I had to manually force GitHub to rerun the CI pipeline in order to get the codecov checks to show. I'm guessing there may have been a transient issue that prevented the checks from being performed. We should continue to keep an eye on this...I wonder if there is a way to enforce that this check happens...

**Comment by Shaumik-Ashraf** [comment]
> Side Note:
> 
> I had to manually force GitHub to rerun the CI pipeline in order to get the codecov checks to show. I'm guessing there may have been a transient issue that prevented the checks from being performed. We should continue to keep an eye on this...I wonder if there is a way to enforce that this check happens... (or maybe we do want the GitHub action too?)

I actually see the codecov checks whenever I push to the branch; after the build, test, and lint actions codcov/patch and codecov/project results just automatically get added. Maybe only the assignee can view them?

**Comment by radamson** [comment]
@Shaumik-Ashraf They are visible to everyone and I can see them on or other open PRs (even without having to reset). I had already reset the checks right after reviewing, otherwise we could have tested to make sure. Next time...

**Pull request merged**